js/views/userPage/UserPage.js

```diff
@@ -48,9 +50,12 @@ export default class extends baseVw {
       });
     }
 
-    this.listenTo(this.model, 'change', () => this.render());
-    this.listenTo(this.model.get('avatarHashes'), 'change', () => this.render());
-    this.listenTo(this.model.get('headerHashes'), 'change', () => this.render());
+    // todo: This is way too elaborate of a view to re-render (at least at an unexpected / random
+    // time). When we need to update UI based off of async changes, we need to either display
+    // some indicator and let the user refresh -or- if the changes are simple enough, make them
+    // with jQuery. In this case, I think jQuery would be simple enough.
+    // this.listenTo(this.model.get('avatarHashes'), 'change', () => this.render());
```

Comment:
@jjeffryes Had to comment out this code. It does unpleasant things to the store and would to the other tabs as we start to build them out.